lightning/src/util/events.rs

```diff
@@ -612,7 +616,7 @@ impl Writeable for Event {
 				// We never write out FundingGenerationReady events as, upon disconnection, peers
 				// drop any channels which have not yet exchanged funding_signed.
 			},
-			&Event::PaymentReceived { ref payment_hash, ref amount_msat, ref purpose } => {
+			&Event::PaymentReceived { ref payment_hash, ref amount_msat, ref purpose, receiver_node_id } => {
```

Comment:
nit: receiver_node_id doesn't need an explicit `ref` here, but may be nice for consistency.